package/network/config/ltq-vdsl-app/src/src/dsl_cpe_ubus.c

```diff
@@ -161,7 +161,7 @@ static inline void m_array(const char *id, const uint8_t *value, uint8_t len) {
 	void *c = blobmsg_open_array(&b, id);
 
 	for (uint8_t i = 0; i < len; ++i)
-		blobmsg_add_u16(&b, "", value[i]);
```

Comment:
I think u16 is fine here because the values are 8 bit unsigned -> u16 provides enough room

Reply:
This is still pending
In my other comment I suggested `git commit --amend`: for this to pick up changed files you also need to give the file/path at the end (so `git commit --amend package/network/...`)

Reply:
@xdarklight can we merge this? seems ready to me

Reply:
I still think that this specific change (`blobmsg_add_u16` -> `blobmsg_add_u64`) is not needed because `m_array()` is not used for the re-transmit counters.
If it's needed then the commit message should be updated to reflect "why" (or even better: create another commit with an according message)

Reply:
Did a new commit to fix this.